src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -58,8 +58,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Mark that this targets file supports package download. -->
     <PackageDownloadSupported>true</PackageDownloadSupported>
     <!-- Flag if the Central package file is enabled -->
-    <EnableCentralPackageVersions Condition="'$(EnableCentralPackageVersions)' == ''">false</EnableCentralPackageVersions>
-    <_CentralPackageVersionsEnabled Condition="'$(EnableCentralPackageVersions)' == 'true' AND '$(CentralPackageVersionsFileImported)' == 'true'">true</_CentralPackageVersionsEnabled>
+    <ManagePackageVersionsCentrally Condition="'$(ManagePackageVersionsCentrally)' == ''">false</ManagePackageVersionsCentrally>
```

Comment:
Do not set this to false.

Reply:
@nkolev92, why to not set to false? The goal is to set the default to false.

Reply:
Properties are not booleans. They are strings. 

The good practice is to check whether something is set to true only. 
If something is opt-in only, an empty value is ok.

Your code should always only check `if true` and continue.

Reply:
Updated.